packages/protocol/contracts/tokenvault/ERC20Vault.sol

```diff
@@ -234,8 +230,7 @@ contract ERC20Vault is
         if (token == address(0)) revert VAULT_INVALID_TOKEN();
 
         if (amount > 0) {
-            if (isBridgedToken[token] || token == resolve("taiko_token", true))
-            {
+            if (isBridgedToken[token]) {
```

Comment:
TaikoToken (now) has no public burn function.
(I think special handling is somewhat necessary for that.)

Reply:
@dantaik I think TKO can be a bridged token so effectively this (ERC20Vault.sol line 234) will try to burn it -> But has no burn function, no ? 🤔

Reply:
Ah so, we are not deploying TaikoToken on L2 ourselves ? 🤔 but let it be “bridged” via the BridgedERC20 ?
If so it is fine, then no comment!